cpu/sam0_common/periph/cpuid.c

```diff
@@ -33,6 +34,12 @@
 
 void cpuid_get(void *id)
 {
+#ifdef HAS_FACTORY_BLCOK
```

Comment:
Typo, it should be `HAS_FACTORY_BLOCK` instead

Reply:
done